windows_install.ps1

```diff
@@ -0,0 +1,48 @@
+#Requires -Version 5
+
+$old_erroractionpreference = $erroractionpreference
```

Comment:
what are these flags for?
I don't see where you use it.

Reply:
thanks, removed the `$old_erroractionpreference`
regarding `$erroractionpreference`, its an env var used by powershell
https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_preference_variables?view=powershell-7.2